compiler/src/org.graalvm.compiler.asm.amd64/src/org/graalvm/compiler/asm/amd64/AMD64Assembler.java

```diff
@@ -825,7 +863,7 @@ protected AMD64RRROp(String opcode, int prefix1, int prefix2, int op, boolean ds
         // TEST is documented as MR operation, but it's symmetric, and using it as RM operation is more convenient.
         public static final AMD64RMOp TESTB  = new AMD64RMOp("TEST",               0x84, OpAssertion.ByteAssertion);
         public static final AMD64RMOp TEST   = new AMD64RMOp("TEST",               0x85);
-        // @formatter:on
+        // @formatter:off
```

Comment:
All these `// @formatter:off` comments need to be paired with a `// @formatter:on` comment.